core/rpc/api/src/author/mod.rs

```diff
@@ -83,4 +83,24 @@ pub trait AuthorApi<Hash, BlockHash> {
 		metadata: Option<Self::Metadata>,
 		id: SubscriptionId
 	) -> Result<bool>;
+
+	/// Watch multiple extrinsics (own or from network)
+	#[pubsub(
+		subscription = "author_extrinsicTrack",
+		subscribe,
+		name = "author_trackExtrinsic"
```

Comment:
```suggestion
		name = "author_watchExtrinsic"
```

Reply:
I'd prefer this name, even if unsubscribe is going to be unintuitively named differently.